lib/webistrano/deployer.rb

```diff
@@ -270,33 +226,57 @@ def self.type_cast(val)
         end
       end
     end
-    
+
     def self.cvs_root_defintion?(val)
       val.index(':') == 0 && val.scan(":").size > 1
     end
-    
-    # override in order to use DB logger 
+
+    # override in order to use DB logger
     def handle_error(error) #:nodoc:
       case error
       when Net::SSH::AuthenticationFailed
         logger.important "authentication failed for `#{error.message}'"
-      when Capistrano::Error
-        logger.important(error.message)
-      else 
+      # when Capistrano::Error
+      #   logger.important(error.message)
```

Comment:
Looks like we want to use the "else" logic, even in this case — do we really want Capistrano errors to produce backtraces?